tests/system/marathon_pods_tests.py

```diff
@@ -102,11 +101,13 @@ def test_create_pod_with_private_image():
         common.delete_secret(secret_name)
 
 
-@shakedown.dcos_1_9
-@pytest.mark.usefixtures("wait_for_marathon_and_cleanup", "events_to_file")
-def test_event_channel_for_pods():
+@shakedown.dcos_1_9 # NOQA
```

Comment:
why NOQA for this?

Reply:
Good question. Flake8 was complaining. I should ignore a specific rule. The issue was about a method parameter.

Reply:
It would be a redefinition pf the fixture.